### ISSUE TITLE
Need your help to verify the feature to manage GitHub Access Token using GNOME Keyring and Windows Credential Manager

### ISSUE THREAD

**Issue by suzuki-shunsuke** [aquaproj/aqua#3178]
We have implemented the feature to manage GitHub Access Token using GNOME Keyring and Windows Credential Manager.

- https://github.com/aquaproj/aqua/pull/3174

We have already confirmed this feature works well using macOS Keychain, but we haven't verified this feature using GNOME Keyring and Windows Credential Manager yet.
We usually develop aqua on macOS, so it's hard to verify it on Windows and Linux.
So we really appreciate it if you can test the feature using Windows and Linux!

You can try the feature using aqua [v2.37.0-1](https://github.com/aquaproj/aqua/releases/tag/v2.37.0-1).

```sh
aqua upa v2.37.0-1
```

About the detail of the feature, please see https://github.com/aquaproj/aqua/pull/3174 .

Thank you for your cooperation!

**Comment by nikolay** [comment]
@suzuki-shunsuke It seems that even the pre-release version has the stable version's executable, so, the new command does not work. I tried both using `aqua upa` and just downloading the artifact from GitHub releases and executing it directly.

**Comment by suzuki-shunsuke** [comment]
If `aqua upa v2.37.0-1` doesn't update the version, please check the path of aqua.

```console
$ command -v aqua
/Users/shunsukesuzuki/.local/share/aquaproj-aqua/bin/aqua

$ ls -lh "$(command -v aqua)"
lrwxr-xr-x 1 shunsukesuzuki staff 96 10 18 09:41 /Users/shunsukesuzuki/.local/share/aquaproj-aqua/bin/aqua -> ../internal/pkgs/github_release/github.com/aquaproj/aqua/v2.37.0-1/aqua_darwin_arm64.tar.gz/aqua
```

**Comment by nikolay** [comment]
@suzuki-shunsuke You're right that on macOS only the following binary gets upgraded: ` ~/.local/share/aquaproj-aqua/bin/aqua`, but I don't know where the binary is installed on Windows 11 to test. I looked into the `AppData` folder, but it still showed the old version.

**Comment by suzuki-shunsuke** [comment]
Windows uses hard links instead of symbolic links, but this difference doesn't matter.

This is on GitHub Actions' windows-latest.

https://github.com/aquaproj/aqua-registry/actions/runs/11397633487/job/31713386883

```console
# aqua -v
aqua version 2.37.0-1 (af26af42cf361cb0c3db604bb9cde1940d5d6d68)

# command -v aqua
/c/Users/runneradmin/AppData/Local/aquaproj-aqua/bin/aqua

# aqua token --help
NAME:
   aqua token - Manage a GitHub Access Token in keyring

USAGE:
   aqua token command [command options]

COMMANDS:
   set      Set a GitHub access token in keyring
   help, h  Shows a list of commands or help for one command

OPTIONS:
   --help, -h  show help

# ls /c/Users/runneradmin/AppData/Local/aquaproj-aqua/internal/pkgs/github_release/github.com/aquaproj/aqua/v2.37.0-1/aqua_windows_amd64.zip/
aqua.exe  LICENSE  README.md
```

aqua is installed in `AppData/Local/aquaproj-aqua/internal/pkgs/github_release/github.com/aquaproj/aqua`.

**Comment by suzuki-shunsuke** [comment]
Hmm. I tried the feature on GitHub Actions windows-latest, but it looks weird.

`aqua token set` succeeds, but seems that the feature doesn't work as expected.
I ran `aqua init` and checked the debug log.
`aqua init` should call GitHub API to get the latest release and should output the debug log about the feature, but the debug log wasn't outputted.
I'm not sure why. 🤔

https://github.com/aquaproj/aqua/blob/508cc3f7bc927daae2675065daaaf333778a8480/pkg/controller/initcmd/init.go#L53

https://github.com/aquaproj/aqua/blob/508cc3f7bc927daae2675065daaaf333778a8480/pkg/github/github.go#L116-L119

https://github.com/aquaproj/aqua/blob/508cc3f7bc927daae2675065daaaf333778a8480/pkg/github/github.go#L92

**Comment by nikolay** [comment]
Same thing, the version is the old one still. Anyway, I will download the executable and try it.

**Comment by nikolay** [comment]
By the way, the token does not work for me on macOS.

**Comment by suzuki-shunsuke** [comment]
Close this issue as we confirmed it worked well with mkouhei.

https://github.com/aquaproj/aqua/pull/3174#issuecomment-2684209386
